$NetBSD: patch-Makefile,v 1.6 2023/05/03 20:13:16 nikita Exp $

Do not pick pkgsrc libtool on Darwin.
Use correct installation dirs.
Build the miniupnpc.pc file

--- Makefile.orig	2022-10-20 00:49:03.000000000 +0200
+++ Makefile	2023-05-03 22:07:39.354561821 +0200
@@ -15,7 +15,7 @@
 
 ifneq (, $(findstring darwin, $(OS)))
 JARSUFFIX=mac
-LIBTOOL ?= $(shell which libtool)
+LIBTOOL ?= /usr/bin/libtool
 endif
 ifneq (, $(findstring linux, $(OS)))
 JARSUFFIX=linux
@@ -157,10 +157,11 @@
 INSTALLDIRINC = $(INSTALLPREFIX)/include/miniupnpc
 INSTALLDIRLIB = $(INSTALLPREFIX)/$(LIBDIR)
 INSTALLDIRBIN = $(INSTALLPREFIX)/bin
-INSTALLDIRMAN = $(INSTALLPREFIX)/share/man
+INSTALLDIRMAN = $(INSTALLPREFIX)/$(PKGMANDIR)
 PKGCONFIGDIR = $(INSTALLDIRLIB)/pkgconfig
 
 FILESTOINSTALL = $(LIBRARY) $(EXECUTABLES)
+FILESTOINSTALL += $(BUILD)/miniupnpc.pc
 ifeq (, $(findstring amiga, $(OS)))
 FILESTOINSTALL += $(SHAREDLIBRARY) $(BUILD)/miniupnpc.pc
 endif
@@ -244,7 +245,7 @@
 	cp $< $<.bak
 	sed 's/\(.*MINIUPNPC_API_VERSION\s\+\)[0-9]\+/\1$(APIVERSION)/' < $<.bak > $<
 
-install:	updateversion $(FILESTOINSTALL)
+install:	$(FILESTOINSTALL)
 	$(INSTALL) -d $(DESTDIR)$(INSTALLDIRINC)
 	$(INSTALL) -m 644 $(HEADERS) $(DESTDIR)$(INSTALLDIRINC)
 	$(INSTALL) -d $(DESTDIR)$(INSTALLDIRLIB)
